api/v1alpha1/entandoappv2_types.go

```diff
@@ -108,6 +108,7 @@ type EntandoAppV2Status struct {
 // An EntandoAppV2 deploys the components required to upgrade an Entando App. The server side
 // components that are deployed include the Entando App Engine, the Entando Component Manager,
 // the Entando App Builder, and the user facing application.
+// +kubebuilder:printcolumn:name="Ready",type="string",JSONPath=`.status.conditions[?(@.type=="Ready")].status`,description="state of EntandoApp"
```

Comment:
adds `additionalPrinterColumns` to CRD, in this way it's easier to see its evolution with kubectl